tests/specs/__snapshots__/tests.spec.ts.snap

```diff
@@ -25,3 +25,5 @@ exports[`tests produce and consume 3`] = `
 `;
 
 exports[`tests produce and consume 4`] = `{}`;
+
+exports[`tests should throw if failed to produce 1`] = `[TypeError: fetch failed]`;
```

Comment:
do we use withThrow in the tests?
what is the returned status code?
because its `fetch` call I think its better to snapshot the response

Reply:
Yes we do, the returned status code is 500 and I have no idea how to make the snapshot better